org.adempiere.server/resources/templates/idempiere/styles/template.css

```
@@ -57,13 +79,73 @@ body {
+/* Responsive (fluid) brand title / subtitle */
+.brand-block {
+    min-width: 0;
+    flex: 1 1 auto;
+    line-height: 1.2;
+}
+.brand-title {
+    font-size: clamp(1.05rem, 4.2vw, 1.75rem);
+    font-weight: 500;
+    color: var(--theme-text-muted) !important;
+    overflow-wrap: break-word;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)